### PR TITLE
Implemented testing for Enumerable Class

### PR DESCRIPTION
**Pull request by Zappat0n**
We created different tests with Rspec for the methods added to the Enumerable Class.
The methods tested are the following:
- my_each
- my_each_with_index
- my_select
- my_all?
- my_any?
- my_none?
- my_count
- my_inject
- my_map

### CODE REVIEW

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Comment by Zappat0n** [comment]
Thank you very much for your review.

**Pull request merged**
